### ISSUE TITLE
Tabs Not Respected After Update

### ISSUE THREAD

**Issue by netpoetica** [progre/tslint-config-airbnb#16]
I think this is likely a TSLint issue, but, after updating tslint-config-airbnb from 1x to 5x, all of a sudden TSLint (5.5.0) is telling me that all of my indentation should be spaces, even though my config looks like this:

```
{
	"extends": [
		"tslint-react",
		"tslint-config-airbnb"
	],
	"rules": {
		"indent": [true, "tabs"],
		"ter-indent": [true, "tabs"]
	}
}
```

I did not change version of TSLint or TSLint-react - only tslint-config-airbnb, and all of a sudden, my rules are no longer respected. Just wondering if you have any thoughts on what might cause this.

I run TSLint like so:

```
node_modules/.bin/tslint -c tslint.json 'src/**/*.ts*'
```

**Comment by netpoetica** [comment]
If it helps any, I have some real strange output from the linter:

```
ERROR: ... Expected indentation of NaN spaces but found 1 tab.
ERROR: ... Expected indentation of NaN spaces but found 0.
ERROR: ... Expected indentation of 0 spaces but found 1 tab.
ERROR: ... Expected indentation of 0tabs spaces but found 2 tabs.
```

**Comment by netpoetica** [comment]
As I expected, this seems not to be an error with tslint-config-airbnb. My config had "tabs" instead of "tab" because the tslint documentation says "tabs". See: https://github.com/palantir/tslint/issues/3027#issuecomment-314472088
